data/capture_config.yaml

```diff
@@ -0,0 +1,26 @@
+spoof_regex: .*
+ntlm_challenge: "1122334455667788"
+ntlm_domain: anonymous
+services:
+  DRDA: off
+  FTP: on
+  HTTP: on
```

Comment:
To future proof this API, what are your thoughts on making services be an array of hashes, that way we can extend this config with extra fields etc in the future:
```yaml
services:
- type: HTTP
  enabled: true
```

i.e. an example of multiple http services with separate datastore values:
```yaml
services:
- type: HTTP
  enabled: true
  datastore:
      lport: 5000
- type: HTTP
  enabled: true
  datastore:
      lport: 8000
```